src/main/java/org/isf/medicalinventory/gui/InventoryBrowser.java

```diff
@@ -41,14 +41,7 @@
 import java.util.GregorianCalendar;
 import java.util.List;
 
-import javax.swing.JButton;
-import javax.swing.JComboBox;
-import javax.swing.JFrame;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-import javax.swing.JScrollPane;
-import javax.swing.JTable;
-import javax.swing.SwingConstants;
+import javax.swing.*;
```

Comment:
The project standards require that all imports are enumerated; no package level imports allowed.  Thanks

Reply:
This has to be fixed per project standards; do not mark as resolved until it is correct.